cmake/FindCharm.cmake

```diff
@@ -385,6 +385,11 @@ target_link_libraries(
   ${CHARM_LIB_ckmain}
   ${CHARM_LIB_conv-static}
   )
+target_compile_definitions(
```

Comment:
It might be a good idea to add a brief comment here what one of the intended uses for this definition is. basically just the long part of the commit message.

Reply:
ok, done